package.json

```diff
@@ -79,15 +79,20 @@
       "packages/*",
       "scripts",
       "packages/fluentui/*"
+    ],
+    "nohoist": [
+      "packages/web-components/typescript",
+      "packages/web-components/typescript/**",
+      "packages/web-components/ts-loader",
+      "packages/web-components/ts-loader/**",
+      "packages/web-components/ts-node",
+      "packages/web-components/ts-node/**",
+      "packages/web-components/mocha",
+      "packages/web-components/mocha/**",
+      "packages/web-components/webpack",
+      "packages/web-components/webpack/**"
     ]
```

Comment:
Should this result in yarn.lock changes?

Reply:
just ran this locally - I don't see any. Should be good!

Reply:
Apparently not. It seems yarn.lock documents the full tree of packages present, but not which ones are installed where (which is probably how this issue was intermittent).